content/posts/studiecirkel-omvarldskunskap2024-01-23.md

```diff
@@ -0,0 +1,24 @@
+title: Studiecirkel omvärldskunskap
```

Comment:
Could we add a hyphen to this filename to for consistency?

Reply:
Fixed.